selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/organization/OrganizationPage.java

```diff
@@ -172,6 +172,7 @@ public void clickAddSuborganizationButton() {
   }
 
   public void waitOrganizationName(String name) {
+    WaitUtils.sleepQuietly(1);
```

Comment:
Please, use static import to improve readability. 
Also, please add comment with explanation why we can't check appearance/disappearance of to certain visual element instead of 1 sec timeout?

Reply:
Can we wait for non-empty attribute and return it then?
It will avoid using `WaitUtils.sleepQuietly(1);`